common/app/model/dotcomrendering/pageElements/PageElement.scala

```diff
@@ -1263,7 +1265,14 @@ object PageElement {
       case Vine =>
         element.vineTypeData
           .map(d =>
-            VineBlockElement(d.html, containsThirdPartyTracking(element.tracking), Some(d.source), d.sourceDomain),
+            VineBlockElement(
+              getIframeSrc(d.html.getOrElse("")).getOrElse(""),
```

Comment:
Is there a better way to get around d.html being an option?

Reply:
Yes. Let me show you in a moment...

Reply:
https://github.com/guardian/frontend/pull/23559